builds/psvita/Makefile

```diff
@@ -32,10 +32,10 @@ CFLAGS  = -Wl,-q -O3 -DPSP2 -DHAVE_LIBSPEEXDSP \
 		-I$(VITASDK)/arm-vita-eabi/include/liblcf \
 		-I$(VITASDK)/arm-vita-eabi/include/pixman-1
```

Comment:
This line needs a "\" at the end.
When Github Desktop support "ammend commit" you can tick that checkbox to integrate it into this commit.